pybamm/__init__.py

```diff
@@ -52,7 +52,13 @@
 )
 from .logger import logger, set_logging_level, get_new_logger
 from .settings import settings
-from .citations import Citations, citations, print_citations
+try:
+    import pybtex
+
+    if pybtex is not None:
```

Comment:
You should use importlib to check if pybtex is installed -
```suggestion
    if importlib.util.find_spec("pybtex") is not None:
```